setup.py

```diff
@@ -45,20 +45,19 @@ def read(fname):
 
 setup(
     name='zodbpickle',
-    version='4.1.2.dev0',
+    version='4.2.dev0',
     description='Fork of Python 3 pickle module.',
     author='Python and Zope Foundation',
     author_email='zodb-dev@zope.dev',
     url='https://github.com/zopefoundation/zodbpickle',
-    license='PSFL 2 and ZPL 2.1',
+    license='PSFL 2 and ZPL-2.1',
     long_description=README,
     classifiers=[
         'Development Status :: 5 - Production/Stable',
         'License :: OSI Approved :: Zope Public License',
         'License :: OSI Approved :: Python Software Foundation License',
         'Programming Language :: Python',
         'Programming Language :: Python :: 3',
-        'Programming Language :: Python :: 3.8',
```

Comment:
Also add 3.14 here

Reply:
We usually only add it once the final Python version is released as it is only preliminary support for now.